index.js

```diff
@@ -84,7 +84,10 @@ function forwardResponseToApiGateway(server, response, context) {
             const body = bodyBuffer.toString(isBase64Encoded ? 'base64' : 'utf8')
             const successResponse = {statusCode, body, headers, isBase64Encoded}
 
-            context.succeed(successResponse)
+            context.succeed(successResponse);
+            if (server && server.close) {
```

Comment:
I realize this is closed, but I am dealing with the same. Thoughts using something like:
`if(process.env.SERVERLESS_LOCAL) {
 ....
}
`